### PR TITLE
Dimitri/ch12364/madrill ajouter la messagerie copol publier

### PR DESCRIPTION
**Pull request by ottaviano**

### CODE REVIEW

File: src/Admin/Designation/CandidateAdmin.php [hunk 1]
```diff
@@ -119,12 +119,9 @@ protected function configureDatagridFilters(DatagridMapper $filter)
             ->add('quality', ChoiceFilter::class, [
                 'show_filter' => true,
                 'label' => 'Qualité',
-                'field_type' => ChoiceType::class,
+                'field_type' => TerritorialCouncilQualityEnum::class,
```
Comment (illusionOfParadise) [inline src/Admin/Designation/CandidateAdmin.php:122]: `field_type` is not correct.

Comment (ottaviano) [reply src/Admin/Designation/CandidateAdmin.php:122]: nice shot 🔫 it should be `TerritorialCouncilQualityChoiceType`

**Review by kevin-verschaeve — APPROVED** [review]

**Comment by ottaviano** [comment]
[ch12477]

**Comment by ottaviano** [comment]
[ch12299]

**Pull request merged**